README.rst

```diff
@@ -2,5 +2,5 @@
 github-experiments
 ==================
 
-How does github behave?
+How does github behave when you create a PR?
```

Comment:
I'd like to point out in a commit comment that this question isn't specific enough.